index.js

```diff
@@ -328,6 +330,8 @@ module.exports = {
     'max-depth': 1,
     // specify the maximum length of a line in your program
     'max-len': [1, 80, 2],
+    // enforce a maximum file length
+    'max-lines': 0,
```

Comment:
Does `'max-lines': 0` mean there is no maximum, or is this saying the max number of lines is 0? If means the same as it reads, it might be a typo.